Package.swift

```diff
@@ -12,9 +12,9 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/WeTransfer/Mocker.git", .upToNextMajor(from: "2.1.0")),
-//        .package(path: "../../Forks/octokit.swift"),
-//        .package(url: "https://github.com/nerdishbynature/octokit.swift", .upToNextMajor(from: "0.10.1")),
-        .package(url: "https://github.com/AvdLee/octokit.swift", .branch("master")),
+        /// Temporarily pointing to the WeTransfer fork of octokit.swift. When a new version is released, we should go back pointing to
```

Comment:
Can we add an explanation as to _why_, not _what_? What in "a new" version would resolve this temporary workaround?

Reply:
Addressed in https://github.com/WeTransfer/GitBuddy/pull/90/commits/b254bc1ecfb014a94b92088432a04651d6337fb6.